build.gradle

```diff
@@ -63,9 +62,6 @@ dependencies {
     compileOnly fg.deobf("mezz.jei:jei-${minecraft_version}:${jei_version}:api")
     runtimeOnly fg.deobf("mezz.jei:jei-${minecraft_version}:${jei_version}")
 
-    compileOnly fg.deobf("mcp.mobius.waila:Hwyla:${hwyla_version}_${minecraft_release}.2:api")
-    //runtimeOnly fg.deobf("mcp.mobius.waila:Hwyla:${hwyla_version}_${minecraft_release}.2")
```

Comment:
1.16 still uses HWYLA, so this should not be removed.

Reply:
this isn't removed, i just chaged it to jade's api which is same as hwyla for 1.16

Reply:
So the Jade API is absolutely identical to HWYLA? Have you checked that both Jade and HWYLA work still?

Reply:
yes